### PR TITLE
Fb auth connected

### LINKED ISSUE #60: Facebook OAuth

### PR DESCRIPTION
**Pull request by boyajay**
#### What's this PR do?

Successfully connects with FB, adds/updates users on the DB, and sends information to the client.
Create structure of user's client side, still incomplete and requires testing.
#### Any background context you want to provide?

FB returns user information, which is then sent back to the client and will be stored in the state. The token created will be stored in the window's sessionState.
#### What are the relevant tickets/issues?

Closes #70, closes #60 
#### Where should the reviewer start?

user related functions on the client-side are in auth/authAction.js
Many server side files needed minor tweaks to massage in the FB auth.
#### Are there tests written yet? How should this be manually tested?

tested manually.
#### Screenshots (if appropriate)

screen shot 2016-03-18 at 11 47 53 pm
#### Questions:
- Does this add new dependencies?
  yes, isomorphic-fetch
- Does our documentation need to be updated?
  no

### CODE REVIEW

File: client/auth/authActions.js [hunk 1]
```diff
@@ -33,3 +32,47 @@ export const loginError = (message) => ({
...
+  };
+
+export const login = () =>
+  dispatch => {
+    dispatch(requestLogin());
+
+    const userRequest = new Request('/api/users', {
+      method: 'get',
+      headers: {
+        user: window.sessionStorage,
```
Comment (leranf) [inline client/auth/authActions.js]: should be window.sessionStorage.token

Comment (abuddiga) [inline client/auth/authActions.js]: token: window.sessionStorage.getItem('com.tripsApp');

File: client/auth/authActions.js [hunk 2]
```diff
@@ -33,3 +32,47 @@ export const loginError = (message) => ({
 export const requestLogout = () => ({
   type: LOGOUT_REQUEST,
 });
+// place your api calls HURRS
+export const logout = () =>
+  dispatch => {
+    dispatch(requestLogout());
+    window.sessionStorage.removeItem('com.tripsApp');
+    dispatch(routeActions.push('/home'));
```
Comment (leranf) [inline client/auth/authActions.js]: change route to /

File: server/config/routes.js [hunk 3]
```diff
@@ -5,7 +5,8 @@ const destinationsController = require('../destinations/destinationsController')
 const eventsController = require('../events/eventsController');
 const tagsController = require('../tags/tagsController');
 const tripsController = require('../trips/tripsController');
-const userController = require('../votes/votesController');
+const userController = require('../users/usersController');
+const user = require('../users/users');
```
Comment (abuddiga) [inline server/config/routes.js]: Capitalize User

**Comment by abuddiga** [comment]
:+1:

**Pull request merged**